docs/reference/api/client/_index.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: "Client (Kn)"
```

Comment:
Prefix with "Knative" like the other titles?

Or you can just get rid of the duplicate `linkTitle` since its the same